entrypoint.sh

```
@@ -185,16 +185,24 @@ if [ "$claude_rc" -ne 0 ] && [ "$claude_rc" -ne 143 ]; then exit "$claude_rc"; f
 if [ -n "${CLAUDIO_RESULT_FILE}" ] && [ -s "${CLAUDIO_LOG_FILE:-}" ]; then
   echo "=== Evaluating task result ==="
 
-  if ! tail -c "${CLAUDIO_RESULT_MAX_CHARS:-50000}" "${CLAUDIO_LOG_FILE}" | \
+  local eval_output
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)